frontend/src/host_orchestrator/main.go

```diff
@@ -81,45 +81,48 @@ func start(starters []func() error) {
 	wg.Wait()
 }
 
-func newOperatorProxy(port string) *httputil.ReverseProxy {
-	if port == "" {
+func newOperatorProxy(port int) *httputil.ReverseProxy {
+	if port <= 0 {
 		log.Fatal("The host orchestrator requires access to the operator")
 	}
-	operatorURL, err := url.Parse(fmt.Sprintf("http://127.0.0.1:%s", port))
+	operatorURL, err := url.Parse(fmt.Sprintf("http://127.0.0.1:%d", port))
 	if err != nil {
-		log.Fatalf("Invalid operator port (%s): %v", port, err)
+		log.Fatalf("Invalid operator port (%d): %v", port, err)
 	}
 	return httputil.NewSingleHostReverseProxy(operatorURL)
 }
 
 func main() {
-	httpPort := fromEnvOrDefault("ORCHESTRATOR_HTTP_PORT", DefaultHttpPort)
-	httpsPort := fromEnvOrDefault("ORCHESTRATOR_HTTPS_PORT", "")
-	tlsCertDir := fromEnvOrDefault("ORCHESTRATOR_TLS_CERT_DIR", DefaultTLSCertDir)
-	certPath := filepath.Join(tlsCertDir, "cert.pem")
-	keyPath := filepath.Join(tlsCertDir, "key.pem")
-	cvdUser := fromEnvOrDefault("ORCHESTRATOR_CVD_USER", "")
-	operatorPort := fromEnvOrDefault("OPERATOR_HTTP_PORT", "")
-
-	abURL := fromEnvOrDefault("ORCHESTRATOR_ANDROID_BUILD_URL", defaultAndroidBuildURL)
-	cvdBinAndroidBuildID := fromEnvOrDefault("ORCHESTRATOR_CVDBIN_ANDROID_BUILD_ID", defaultCVDBinAndroidBuildID)
-	cvdBinAndroidBuildTarget := fromEnvOrDefault("ORCHESTRATOR_CVDBIN_ANDROID_BUILD_TARGET", defaultCVDBinAndroidBuildTarget)
-	imRootDir := fromEnvOrDefault("ORCHESTRATOR_CVD_ARTIFACTS_DIR", defaultCVDArtifactsDir)
+	httpPort := flag.Int("http_port", 1080, "Port to listen on for HTTP requests.")
+	httpsPort := flag.Int("https_port", -1, "Port to listen on for HTTPS requests.")
+	tlsCertDir := flag.String("tls_cert_dir", defaultTLSCertDir, "Directory with the TLS certificate.")
+	cvdUser := flag.String("cvd_user", "", "User to execute cvd as.")
+	operatorPort := flag.Int("operator_http_port", 2080, "Port where the operator is listening.")
+	abURL := flag.String("android_build_url", defaultAndroidBuildURL, "URL to an Android Build API.")
+	cvdBinAndroidBuildID := flag.String("cvd_build_id", defaultCVDBinAndroidBuildID, "Build ID to fetch the cvd binary from.")
+	cvdBinAndroidBuildTarget := flag.String("cvd_build_target", defaultCVDBinAndroidBuildTarget, "Build target to fetch the cvd binary from.")
+	imRootDir := flag.String("cvd_artifacts_dir", defaultCVDArtifactsDir, "Directory where cvd will download android build artifacts to.")
+
+	flag.Parse()
+
+	certPath := filepath.Join(*tlsCertDir, "cert.pem")
+	keyPath := filepath.Join(*tlsCertDir, "key.pem")
+
 	imPaths := orchestrator.IMPaths{
-		RootDir:          imRootDir,
-		CVDToolsDir:      imRootDir,
-		ArtifactsRootDir: filepath.Join(imRootDir, "artifacts"),
-		RuntimesRootDir:  filepath.Join(imRootDir, "runtimes"),
+		RootDir:          *imRootDir,
```

Comment:
nit: using the Var() functions we can avoid the `*` resulting in a more consistent reading/view of the code.

Reply:
discussed offline, keeping like this for now